src/Makefile

```diff
@@ -271,6 +271,10 @@ ifeq ($(MALLOC),jemalloc)
 	FINAL_LIBS := ../deps/jemalloc/lib/libjemalloc.a $(FINAL_LIBS)
 endif
 
+ifeq ($(HLL_HASH),wyhash)
+	FINAL_CFLAGS+= -DUSE_WYHASH
```

Comment:
Could you please enlist all the test cases that you're planning or already performed to validate this change?. 
How are we ensuring that this new hash function is hardened properly for all our uses cases?